conf/flight_plans/tudelft/oneloop_cyberzoo.xml

```diff
@@ -8,7 +8,16 @@
     #include "modules/radio_control/radio_control.h"
     #include "modules/ahrs/ahrs.h"
     #include "modules/gps/gps.h"
+    #include "modules/rotwing_drone/rotwing_state.h"
```

Comment:
Please note that `modules.h` is visible by the flight plan, so it should not be necessary to include modules' header files by hand